docs/ios/open-source/features/capture-services/ui-view-controller.md

```diff
@@ -46,6 +50,12 @@ The service will also generate child spans measuring several steps in the proces
 If the `UIViewController` disappears before it finishes loading, the spans status will be set to `.error` with the `.userAbandon` code.
 :::
 
+:::warning
+If you're using the Embrace Dashboard with your app, you might need to contact and Embrace representative to enable this feature through the remote configuration.
+
+If you're not using the Embrace Dashboard, you can enable this feature by passing a custom `EmbraceConfigurable` with `isUiLoadInstrumentationEnabled` set to true when [initializing the SDK](/docs/ios/open-source/integration/embrace-options/).
```

Comment:
what version is `isUiLoadInstrumentationEnabled` available when implementing `EmbraceConfigurable`? Not seeing it in 6.6.0, so would note here the minimum version in which to see it.

Reply:
Wasn't released yet.
Will probably be in 6.6.1.